.gitignore

```diff
@@ -32,3 +32,16 @@ DerivedData
 # CocoaPods
 Pods
 *.lock
+
+.idea/.name
+.idea/encodings.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/runConfigurations/iOS.xml
+.idea/runConfigurations/Tests.xml
+.idea/vcs.xml
+.idea/Viewer.iml
+.idea/workspace.xml
+.idea/xcode.xml
+Demo.xcodeproj/project.pbxproj
+Demo.xcodeproj/project.pbxproj
```

Comment:
`Demo.xcodeproj/project.pbxproj` is needed in order to run the project. I was only talking about that, hehe. The idea ones should probably go in the global gitignore